test/torchaudio_unittest/librosa_compatibility_test.py

```diff
@@ -180,19 +174,30 @@ def assert_compatibilities(self, n_fft, hop_length, power, n_mels, n_mfcc, sampl
         out_torch = spect_transform(sound).squeeze().cpu()
         self.assertEqual(out_torch, torch.from_numpy(out_librosa), atol=1e-5, rtol=1e-5)
 
-        # test mel spectrogram
+    def assert_compatibilities_mel_spectogram(self, n_fft, hop_length, n_mels, norm, sound, sample_rate, sound_librosa):
         melspect_transform = torchaudio.transforms.MelSpectrogram(
             sample_rate=sample_rate, window_fn=torch.hann_window,
-            hop_length=hop_length, n_mels=n_mels, n_fft=n_fft)
+            hop_length=hop_length, n_mels=n_mels, n_fft=n_fft, norm=norm)
         librosa_mel = librosa.feature.melspectrogram(
             y=sound_librosa, sr=sample_rate, n_fft=n_fft,
-            hop_length=hop_length, n_mels=n_mels, htk=True, norm=None)
+            hop_length=hop_length, n_mels=n_mels, htk=True, norm=norm)
         librosa_mel_tensor = torch.from_numpy(librosa_mel)
         torch_mel = melspect_transform(sound).squeeze().cpu()
         self.assertEqual(
             torch_mel.type(librosa_mel_tensor.dtype), librosa_mel_tensor, atol=5e-3, rtol=1e-5)
 
-        # test s2db
+    def assert_compatibilities_s2db(self, n_fft, hop_length, power, n_mels, norm, sound, sample_rate, sound_librosa):
+        spect_transform = torchaudio.transforms.Spectrogram(
```

Comment:
The issue talked about splitting out MelSpectrogram was assert_compatibilities, but it was being reused for the s2db and MFCC checks, so I decided to split it all out into individual a bunch of individual functions with some repeated set up work. Let me know what you think.

Reply:
Yeah, for the simplicity of task, I was going to suggest recreating the variable used by tests following MelSpectrogram, but splitting all the tests is much better for maintainability.